src/Raven.Client/Data/DatabaseStatistics.cs

```diff
@@ -79,6 +79,8 @@ public class IndexInformation
         public IndexType Type { get; set; }
 
         public DateTime? LastIndexingTime { get; set; }
+
+        public string OutputReduceToCollection { get; set; }
```

Comment:
I know, but why we do need it here? The idea was to have this as thin as possible.